app/core/src/main/java/com/fsck/k9/KoinModule.kt

```diff
@@ -28,7 +28,7 @@ val mainModule = module {
     single { get<Context>().resources }
     single { get<Context>().contentResolver }
     single { LocalStoreProvider() }
-    single { Contacts.getInstance(get()) }
+    single { Contacts(get()) }
```

Comment:
Can you please use named arguments when editing lines in Koin modules. That makes it a bit easier to see what `get()` will fetch when looking at diffs without an IDE.